x/mongo/driver/topology/connection.go

```diff
@@ -322,10 +324,14 @@ func (c *connection) readWireMessage(ctx context.Context, dst []byte) ([]byte, e
 	if err != nil {
 		// We closeConnection the connection because we don't know if there are other bytes left to read.
 		c.close()
+		message := "incomplete read of message header"
```

Comment:
This same code is repeated below, so let's refactor it into a function like `getReadErrrorMessage(err error) string`.

Reply:
I'm not sure how necessary it would be to refactor this into a function like `getReadErrrorMessage(err error) string`. The original message is slightly different (`incomplete read of message header` vs `incomplete read of full message`), so `getReadErrorMessage` would basically just be one if statement

Reply:
Ah, never mind then. This is fine. Resolving.